src/briefcase/platforms/macOS/xcode.py

```diff
@@ -17,13 +17,15 @@
 
 class macOSXcodeMixin(macOSMixin):
     output_format = "Xcode"
+    supported_host_os_reason = (
+        "macOS applications require the Xcode command line "
+        "tools, which are only available on macOS."
+    )
 
     def verify_tools(self):
-        if self.tools.host_os != "Darwin":
-            raise BriefcaseCommandError(
-                "macOS applications require the Xcode command line tools, "
-                "which are only available on macOS."
-            )
+        if self.tools.host_os not in self.supported_host_os:
+            raise BriefcaseCommandError(self.supported_host_os_reason)
+
```

Comment:
As above, I wonder if this indicates a need for a generic `verify_host()` check

Reply:
Implemented `verify_host()`.